test/object-store/sync/app.cpp

```diff
@@ -4147,12 +4188,12 @@ TEST_CASE("app: base_url", "[sync][app][base_url]") {
         std::string redir_wsurl = util::format("ws%1://%2:%3", use_ssl ? "s" : "", expected_host, expected_port);
 
         auto socket_provider = std::make_shared<HookedSocketProvider>(logger, "some user agent");
-        socket_provider->endpoint_verify_func = [&use_ssl, &expected_host,
-                                                 &expected_port](sync::WebSocketEndpoint& ep) {
-            CHECK(ep.address == expected_host);
-            CHECK(ep.port == expected_port);
-            CHECK(ep.is_ssl == use_ssl);
-        };
+        //        socket_provider->endpoint_verify_func = [&use_ssl, &expected_host,
```

Comment:
meant to be removed?

Reply:
Actually, I think I'll add it back in, since it does verify the correct info is making it down to the sync socket provider.